README.md

```diff
@@ -69,24 +69,24 @@ d’accès aux données ou un bloc RGPD si vous exposez des données personnelle
 des informations à déterminer ensemble (ainsi que les fichiers à modifier dans le code de DataPass) :
 
 1. dans le frontend
-    1. description de l’organisation du formulaire (création de src/pages/NameOfApi.js)
-    2. url du formulaire sur le domaine datapass.api.gouv.fr (src/Routes.js)
-    3. label à afficher pour le service dans la vue liste (src/lib/api.js)
-    4. [optionnel] codes organisation (codes NAF) valides pour votre service (src/lib/index.js L~38)
-    6. [optionnel] une page de présentation hors connection
+   1. description de l’organisation du formulaire (création de src/pages/NameOfApi.js)
+   2. url du formulaire sur le domaine datapass.api.gouv.fr (src/Routes.js)
+   3. label à afficher pour le service dans la vue liste (src/lib/api.js)
+   4. [optionnel] codes organisation (codes NAF) valides pour votre service (src/lib/index.js L~38)
+   5. [optionnel] une page de présentation hors connection
```

Comment:
what convention do you use for linting markdown ?

Reply:
I use CMD+S in VSCode haha

Reply:
💋 my 🍑